pkg/cmd/init.go

```diff
@@ -288,6 +288,7 @@ func certSetup(cfg *config.Config) (*certchains.CertificateChains, error) {
 					"openshift.default.svc.cluster.local",
 					"api." + cfg.DNS.BaseDomain,
 					"api-int." + cfg.DNS.BaseDomain,
+					"10.44.0.0",
```

Comment:
This IP is subject to change and its stored as part of the configuration:
```suggestion
					cfg.ApiServer.AdvertiseAddress,
```